csrankings-k.csv

```diff
@@ -337,6 +337,7 @@ Kayla Booth,University of Pittsburgh,http://www.sci.pitt.edu/faculty-and-researc
 Kayla M. Booth,University of Pittsburgh,http://www.sci.pitt.edu/faculty-and-research/faculty-directory/kayla-booth,NOSCHOLARPAGE
 Kayvon Fatahalian,Stanford University,http://graphics.stanford.edu/~kayvonf,ZeG4wDgAAAAJ
 Kazem Taghva,University of Nevada Las Vegas,https://www.unlv.edu/people/kazem-taghva,m4mAgIgAAAAJ
+Kazem Taram,University of Purdue,https://mktrm.github.io,DX4p3bQAAAAJ&hl

```

Comment:
Institution name is "Purdue University". Please remove spurious "&hl" from end of Google Scholar ID.

Reply:
Updated.